Lib9c/TableData/ArenaSheet.cs

```diff
@@ -86,6 +106,25 @@ public bool TryGetChampionshipRound(out RoundData roundData)
                 roundData = Round.FirstOrDefault(x => x.ArenaType.Equals(ArenaType.Championship));
                 return !(roundData is null);
             }
+
+            public bool TryGetSeasonNumber(int round, out int seasonNumber)
```

Comment:
`TryGetSeasonNumber()` is only used for 9c. I think it is better to move to 9c.

Reply:
It's still correct to actually use it only on the frontend.

However, I believe that these kinds of implicit promises should be protocolized, and validating them with unit tests helps the code to be sustainable.

I have added unit tests. What do you think of this part?

Reply:
i agree with @hskim881028

Reply:
@hskim881028 @ipdae I removed this function.